tiledb/common/src/query/read.rs

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Is this something that should be deleted? Looks like it was the start of something that was either deemed unnecessary or done differently.

Reply:
Derp. Deleleted